src/importlinter/domain/imports.py

```diff
@@ -60,29 +63,29 @@ def is_package(self) -> bool:
         raise NotImplementedError
 
 
-class DirectImport(ValueObject):
+class ImportExpression(ValueObject):
```

Comment:
We'll want to add this as a class, rather than replace `DirectImport`, which is useful too.

Reply:
The reason why I removed it was, because no other usage was in the code base. I can add it again

Reply:
ok